data/_main.cfg

```diff
@@ -49,7 +49,7 @@
 #endif
 [/ais]
 
-#ifdef DEBUG_MODE
+#ifdef TEST
```

Comment:
The contents of this #ifdef can be added to the already existing one starting on line 63.

Reply:
Or moved into data/test/_main.cfg